src/screens/Home/Home.styles.js

```diff
@@ -1,9 +1,10 @@
-import { StyleSheet } from 'react-native';
+import { Dimensions, StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
   container: {
     flex: 1,
     justifyContent: 'center',
     alignItems: 'center',
+    width: Dimensions.get('screen').width,
```

Comment:
Why are you adding this to the container style? just curious,`justifyContent` and `alignItems` should be enough.